crates/nu-cmd-extra/src/extra/bits/or.rs

```diff
@@ -69,24 +123,66 @@ impl Command for BitsOr {
                     Span::test_data(),
                 )),
             },
+            Example {
+                description: "Apply logical or to binary data",
+                example: "0x[88 cc] | bits or 0x[42 32]",
```

Comment:
An example that passes the endianness would be nice. Same for the other commands, so that all the functionality can be demonstrated (and tested)